code/game/objects/items/stacks/sheets/sheet_types.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+const {Component, Atom, has_component, chain_func} = require('bluespess');
```

Comment:
I'm pretty sure Atom, has_component, and chain_func are not used anywhere in this file. Please get rid of them. Keep Component though.